src/common-ssh/ssh.c

```diff
@@ -321,6 +321,10 @@ static int guac_common_ssh_authenticate(guac_common_ssh_session* common_session)
     guac_client_log(client, GUAC_LOG_DEBUG,
             "Supported authentication methods: %s", user_authlist);
 
+    /* If auth list is NULL, then authentication has succeeded with NONE */
+    if (user_authlist == NULL)
```

Comment:
If `user_authlist` is `NULL`, what happens to that `guac_client_log()` call above that tries to print the value?

Reply:
Good point.  Perhaps that NULL check should be located elsewhere...

Reply:
Relocated and logged the NONE success.